Program.cs

```diff
@@ -1,16 +1,161 @@
 ﻿// See https://aka.ms/new-console-template for more information
 
+using System.Globalization;
+using System.Runtime.InteropServices.JavaScript;
 using Kristina_Kulich__Application;
+using Kristina_Kulich__Application.Helpers;
+using Kristina_Kulich__Application.CSharpLinq;
 
 class Program
 {
     static void Main(string[] args)
     {
-     new Homework_1().Task_1();   
-     new Homework_1().Task_2();
-     new Homework_1().Task_3();
-     new Homework_1().Task_4();
-     new Homework_1().Task_5();
-     new Homework_1().Task_6();
-     }
+        #region Task1
+
+        int[] integers = new[] { 12, 8, 27, 78, 2, 3, 3, 4, 21, 1 };
+        List<int> divedsby3 = integers.Select(i => i).Where(i => (i % 3 == 0)).ToList();
```

Comment:
what is a purpose of Select(i => i)

Reply:
code is red underlined without it "Cannot resolve method 'Select()', candidates are:   System.Collections.Generic.IEnumerable<TResult> Select<int,TResult>(this System.Collections.Generic.IEnumerable<int>, System.Func<int,int,TResult>) (in class Enumerable)   System.Collections.Generic.IEnumerable<TResult> Select<int,TResult>(this System.Collections.Generic.IEnumerable<int>, System.Func<int,TResult>) (in class Enumerable)"